engine/src/main/java/org/terasology/logic/players/PlayerFactory.java

```diff
@@ -134,15 +134,15 @@ private Vector3f findSpawnPos(Vector3f targetPos, float entityHeight) {
         int targetBlockY = TeraMath.floorToInt(targetPos.y);
         int targetBlockZ = TeraMath.floorToInt(targetPos.z);
         Vector2i center = new Vector2i(targetBlockX, targetBlockZ);
-        for (BaseVector2i pos : SpiralIterable.clockwise(center).maxRadius(3).scale(2).build()) {
+        for (BaseVector2i pos : SpiralIterable.clockwise(center).maxRadius(250).scale(2).build()) {
```

Comment:
3 might be a bit low, but 250 seems too high. Maybe 16 or 32 is a better compromise? What is the default chunk size nowadays?